firefox-ios/Storage/Rust/RustFirefoxSuggestion.swift

```diff
@@ -21,7 +21,14 @@ public enum RustFirefoxSuggestionTelemetryInfo {
 }
 /// A Firefox Suggest search suggestion. This struct is a Swiftier
 /// representation of the Rust `Suggestion` enum.
-public struct RustFirefoxSuggestion {
+public struct RustFirefoxSuggestion: Equatable {
+    public static func == (lhs: RustFirefoxSuggestion, rhs: RustFirefoxSuggestion) -> Bool {
+        return lhs.title == rhs.title &&
+        lhs.url == rhs.url &&
+        lhs.isSponsored == rhs.isSponsored &&
+        lhs.iconImage == rhs.iconImage
```

Comment:
I omitted the telemetryInfo comparison, let me know if you see any issues with this.